marathon/datadog_checks/marathon/marathon.py

```diff
@@ -78,31 +92,23 @@ def refresh_acs_token(self, auth, acs_url, tags=None):
             )
             raise Exception("Got %s when hitting %s" % (r.status_code, acs_url))
 
-    def get_json(self, url, timeout, auth, acs_url, verify, tags=None):
+    def get_json(self, url, acs_url, tags=None):
         if tags is None:
             tags = []
 
-        params = {'timeout': timeout, 'headers': {}, 'auth': auth, 'verify': verify}
-        if acs_url:
-            # If the ACS token has not been set, go get it
-            if not self.ACS_TOKEN:
-                self.refresh_acs_token(auth, acs_url, tags)
-            params['headers']['authorization'] = 'token=%s' % self.ACS_TOKEN
-            del params['auth']
```

Comment:
Any reason you removed that section?

Reply:
Sorry! I misunderstood and thought that the `self.refresh_acs_token` call was only needed on the `get_json` retry.